impl/src/main/java/io/jsonwebtoken/impl/lang/DefaultCollectionMutator.java

```diff
@@ -46,6 +47,33 @@ private boolean doAdd(E e) {
         return this.collection.add(e);
     }
 
+    public M replace(E e) {
```

Comment:
I would expect this to be something like `public M replace(E old, E new)`, explicitly instruction what should be replaced and with what.

Reply:
However, take note of the `Identifiable` logic above.